CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+* Added an `include` field for configuration, which may specify another YAML
```

Comment:
Please give this a `## 0.12.32` header and update the pubspec to `0.12.32-dev`.

Reply:
Done